packages/block-library/src/template-part/edit/placeholder/index.js

```diff
@@ -1,20 +1,25 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 import { useCallback, useState } from '@wordpress/element';
-import { useDispatch } from '@wordpress/data';
+import { useDispatch, useSelect } from '@wordpress/data';
 import { Placeholder, Dropdown, Button } from '@wordpress/components';
-import { blockDefault } from '@wordpress/icons';
 import { serialize } from '@wordpress/blocks';
 import { store as coreStore } from '@wordpress/core-data';
+import { store as editorStore } from '@wordpress/editor';
 
 /**
  * Internal dependencies
  */
 import TemplatePartSelection from '../selection';
 import PatternsSetup from './patterns-setup';
 
+/**
+ * External dependencies
+ */
+import { find } from 'lodash';
```

Comment:
Nit: external deps should go first for consistency.
- External
- WordPress
- Internal

Reply:
makes sense. updated!